.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": [
+    "node-services",
+    "prettier"
+  ],
+  "plugins": [
+    "prettier"
```

Comment:
eslint-plugin-json was brought up because eslint-config-node-services or the wikimedia one had it as a peer dependency. If you think it is worth using, I'd appreciate if you can send a commit configuring it, if the node-services preset or the wikimedia one are not actually configuring it. I've never used it before.